src/pixelflow/core.py

```diff
@@ -208,12 +208,26 @@ def pixelflow(
         else:
             spacing = (1,) * mask.ndim
 
+    if any(val == 0 for val in spacing):
+        raise ValueError("Spacing cannot be zero")
+
     # add warning for spacing and custom functions
     if any(val != 1 for val in spacing) and custom is not None:
         warnings.warn("Spacing may not work as expected for custom functions.")
 
     # If image is YX then use regionprops_table
     if dim_labels == "YX":
+        if features is None:
+            features = (
+                "label",
+                "bbox",
+                "centroid",
+                "area",
+                "major_axis_length",
+                "eccentricity",
+                "orientation",
+                "solidity",
+            )
```

Comment:
Is there a reason to specify these here?

Reply:
At the moment, if you don't supply any features for a 2D image, then you get an error, however a 3D image gives you subset of the possible 3D features (which is the regionprops3D default). So to give consistency I added a set of the common features for 2D, all of which work with isotropic and anisotropic spacing. If there's any particular ones that you think should or shouldn't be included then happy to alter the list.

Reply:
Cool - thanks for the explanation!